app/Models/Server.php

```
@@ -1184,6 +1186,21 @@ public function installDocker()
         return InstallDocker::run($this);
     }
 
+    /**
+     * Validate that required commands are available on the server.
+     *
+     * @return array{success: bool, missing: array<string>, found: array<string>}
+     */
+    public function validatePrerequisites(): array
+    {
+        return ValidatePrerequisites::run($this);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Actions\Server\ValidatePrerequisites' in method 'validatePrerequisites'. (undefined)

(StaticAccess)